merchantapi/src/main/java/fi/hel/verkkokauppa/merchant/model/LocaleModel.java

```diff
@@ -0,0 +1,19 @@
+package fi.hel.verkkokauppa.merchant.model;
```

Comment:
Same comments than in ConfigurationModel.java regarding the @ToString and public fields.